assets/csidriveroperators/azure-disk/08_deployment.yaml

```diff
@@ -52,6 +52,10 @@ spec:
           requests:
             memory: 50Mi
             cpu: 10m
+        volumeMounts:
```

Comment:
@jstuever I'm not sure we need to mount the token here for the operator, as it does not authenticate with Azure afaik. Shouldn't we mount the token into the [controller driver](https://github.com/openshift/azure-disk-csi-driver-operator/blob/f71f1449938a1dde07904dc079a3f36284f8e99a/assets/controller.yaml#L74) instead?